wiki/Community/Internet_Relay_Chat/zh.md

```diff
@@ -1,88 +1,87 @@
+| [XChat](http://xchat.org/) | 输入 `/set irc_conf_mode 1` |
 
-## 基本的 IRC 命令
+其他的 IRC 客户端用户可以参阅对应客户端的文档。
 
-| 描述 | 操作 |
-| :-- | :-- |
-| 加入频道 (例如 \#lobby) | `/join #频道名称` |
-| 离开频道 | `/part` |
-| 忽略某人的发言 | `/ignore 那人的昵称` |
-| 做些动作 | `/me 做些动作` |
+## 常见问题
 
-## 停用加入/退出频道通知
+### 我不能登录
 
-当有人加入或退出频道时，你会收到这样的通知：
+osu!Bancho 使用纯文本身份验证——确保在您的 IRC 客户端设置中没有选择特殊的身份验证模式。
 
-```
-someuser has joined #somechannel
-someuser has quit #somechannel
-```
+或者你可以尝试连接到 `cho.ppy.sh`（你仍然会连接至 osu!Bancho）
 
-尽管在某些低流量的频道中这不是什么，但对于 \#osu 频道来说，频繁的加入和退出通知会让用户难以追踪最近的消息。
+### 我收到了 "Bad Authentication Token" （身份验证令牌）错误
 
-### 操作方法
+1. 检查你使用的密码是否和 [IRC 验证页面](https://osu.ppy.sh/p/irc) 上的一致。
+2. 应使用下划线替代用户名中的空格（例如，`This Username` 应按 `This_Username` 的格式来输入）。
 
-| 客户端 | 操作 |
-| :-- | :-- |
-| [HexChat](https://hexchat.github.io/) | 右键单击某一频道，在设置子选单中，点击"Hide Join/Part Messages"。前往 Settings » Preferences, 在 Chatting » General 中 , 勾选 "Hide join and part Messages" |
-| [ircII](http://www.eterna.com.au/ircii/) | `/ignore * crap` |
-| [Irssi](https://irssi.org) | `/ignore -channels #频道名称 * JOINS PARTS QUITS` |
-| [Weechat](https://weechat.org/) | `/filter add irc_smart_weechat irc.somename.#频道名称 irc_smart_filter *`。**Note:** somename 是你在 Weechat 中定义的 irc 服务器地址。 |
-| [KVIrc](https://www.kvirc.net/) | 参阅[KVIrc 论坛上的信息](http://www.kvirc.ru/forum/?topic=609.0) . |
-| [mIRC](https://www.mirc.com/) | Tools » Options » pick "IRC". Click the "Events..." button. Change the "joins", "parts", "quits", and "nicks" to your desired settings: "In Status" or "Hide" are good options [1](http://web.archive.org/web/20160304201229/http://i.clintecker.com/disable-irc-msgs.html). |
-| [Quassel IRC](https://quassel-irc.org/) | 右键点击聊天窗口，选择 Hide Events » Join/Part/Quit. |
-| [XChat](http://xchat.org/) | Right-click on the tab you want to change. In the submenu of the channelname, there's a toggle-item "Show join/part messages", simply turn this off. Or you type `/set irc_conf_mode 1` [2](http://xchat.org/faq/#q211) to disable the messages throughout the channels. |
+### 我能使用其他的昵称么?
 
-其他的 IRC 客户端用户可以参阅对应客户端的文档。
+不能。
 
-## 常见问题
+### 如何快速的输入别人的用户名？
 
-### 发生了错误 "Bad Authentication Token". 我该怎么办?
+输入他们用户名的前几个字母，然后使用 `Tab` 从列表循环选择。
 
-1. 检查你使用的密码是否和 [IRC Authentication](https://osu.ppy.sh/p/irc) 页面上的一致。
-2. 如果你的用户名中有空格，用下划线替代它 (**This Username** 应该是 **This\_Username**)
+### 为什么有些人的用户名前缀不同？
 
-### 我能使用其他的昵称么?
+IRC 标准有一个 IRC 模式的概念，可以理解为每个用户的权限或状态。在 osu!Bancho 中，有两种模式被用于特殊的用户组
 
-不能，你必须使用你游戏中的用户名。
+- `+`，或者“语音状态”：用户通过外部 IRC 客户端连接
+- `@`，或者“聊天管理员状态”：用户是聊天室管理员（[GMT（全局管理团队）](/wiki/People/The_Team/Global_Moderation_Team) 或者 [NAT（审核评估团队）](/wiki/People/The_Team/Nomination_Assessment_Team)）
 
-### 我在某些人的名字前看到了 voice 状态，那是什么？
+通过游戏或网站连接到 osu!Bancho 的用户将没有用户名前缀。
 
-voice 状态（名字前面有一个“+”）的用户是通过 IRC 客户端连接的用户，（除了 IRC 管理员，他们的名字前有一个“@”）。
+### 有人给我发送了私信，但是他们并没有在频道用户列表中！
 
-名字前没有前缀的用户是通过游戏连接到 osu! Bancho 的用户。
+他们要么在使用[网页聊天](https://osu.ppy.sh/community/chat)或者使用[osu!(lazer)](https://github.com/ppy/osu)来连接 osu!Bancho。
```

Comment:
same as above, not sure what the mention of osu!bancho is about

Reply:
Most of these I think are grammatical reasons.
English uses `connected` directly then ignore the server

Users connected **to osu!Bancho** via osu! client or web site have no prefix.

Reply:
if it's just a grammar thing and it'd make more sense to readers like this, then it's ok. just making sure it wasn't a mistake

Reply:
Then I don't think there will be any problem.
Many players consider bancho as a server that includes chat